src/ray/core_worker/transport/direct_task_transport.cc

```diff
@@ -44,7 +44,7 @@ Status CoreWorkerDirectTaskSubmitter::SubmitTask(TaskSpecification task_spec) {
       // https://docs.google.com/document/d/1EAWide-jy05akJp6OMtDn58XOK7bUyruWMia4E-fV28/edit?usp=sharing
       auto actor_id = task_spec.ActorCreationId();
       auto task_id = task_spec.TaskId();
-      RAY_LOG(INFO) << "Creating actor via GCS actor id = : " << actor_id;
+      RAY_LOG(DEBUG) << "Creating actor via GCS actor id = : " << actor_id;
```

Comment:
I'm not sure about this. The actor/worker creation is not of high frequency and sometimes it helps us to see what happened when the actor didn't come up successfully. Maybe we keep its level unchanged?

Reply:
Hmm yeah I thought there should be the same duplicated logs in the upper layer (like core_worker.cc), but it seems like it doesn't. Let me revert it.